index.js

```diff
@@ -35,9 +35,15 @@ class versionRequest {
           } else {
             req.version = version
           }
+          if (options && options.removeQueryParam) {
```

Comment:
make sure you explicitly check for `options.removeQueryParam === true` otherwise users can accidentally send some string, array or object, even empty and the if condition will still evaluate.